examples/grpc-server-with-prometheus/server/server.go

```diff
@@ -72,8 +72,7 @@ func main() {
 	// Register your service.
 	pb.RegisterDemoServiceServer(grpcServer, demoServer)
 
-	// Initialize all metrics.
-	grpcMetrics.InitializeMetrics(grpcServer)
```

Comment:
this causes publishing over 3k metrics without values, if we publish all variations with custom labels it can reach >10k metrics with no real value

![image](https://user-images.githubusercontent.com/91831719/202290457-780b2974-d8eb-489c-821b-6159def72d49.png)